gsl-sys/src/auto.rs

```diff
@@ -18529,7 +18529,16 @@ extern "C" {
 }
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
-pub struct gsl_multiroot_function_struct;
+pub struct gsl_multiroot_function_struct {
```

Comment:
Just to be sure: did you write this manually or by running the command?

Reply:
I wrote this manually originally because the regen bin produced a lot of changes in `auto.rs`. I'll run the command and use the autogenerated one... just need to add those structs to the `should_strip_struct` function in regen.rs.